imports/plugins/core/dashboard/client/components/index.js

```diff
@@ -1,4 +1,4 @@
 export { default as ActionView } from "./actionView";
 export { default as PackageList } from "./packageList";
-export { default as Toolbar } from "./toolbar";
+export { default as PublishControls } from "./publishControls";
```

Comment:
Relevant comment about this file name change:

- I noticed that the PublishControls component was created as a class with the named `PublishControls`, but in a file named `toolbar.js`. 
- At first, I registered it as toolbar component i.e `registerComponent("Toolbar")` but that failed to render when called as `Components.Toolbar`.  I later saw that there is another component called toolbar, so I renamed the file for PublishControls to be `publishControls.js`